File: VB/PostponeControllerTest.Module/Task.vb
Imports System
Imports System.ComponentModel

Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering

Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Validation

Namespace PostponeControllerTest.Module
    <DefaultClassOptions, ImageName("BO_Task")> _
    Public Class Task
        Inherits BaseObject

        Public Sub New(ByVal session As Session)
            MyBase.New(session)
        End Sub

        Public Property Description() As String
            Get
                Return GetPropertyValue(Of String)("Description")
            End Get
            Set(ByVal value As String)
                SetPropertyValue(Of String)("Description", value)
            End Set
        End Property

        Public Property DueDate() As Date
            Get
                Return GetPropertyValue(Of Date)("DueDate")
            End Get
            Set(ByVal value As Date)
                SetPropertyValue(Of Date)("DueDate", value)
            End Set
        End Property
    End Class
End Namespace

File: CS/PostponeControllerTest.Module/Task.cs
using System;
using System.ComponentModel;

using DevExpress.Xpo;
using DevExpress.Data.Filtering;

using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Validation;

namespace PostponeControllerTest.Module {
    [DefaultClassOptions, ImageName("BO_Task")]
    public class Task : BaseObject {
        public Task(Session session) : base(session) { }

        public string Description {
            get { return GetPropertyValue<string>("Description"); }
            set { SetPropertyValue<string>("Description", value); }
        }

        public DateTime DueDate {
            get { return GetPropertyValue<DateTime>("DueDate"); }
            set { SetPropertyValue<DateTime>("DueDate", value); }
        }
    }   
}
